hooks/useMintCandyMachine.tsx

```diff
@@ -82,14 +82,18 @@ const useMintCandyMachine = (account: string) => {
             setIsUserMinting(true)
             refreshCandyMachineState()
 
+            const provider = new AnchorProvider(new Connection(connection.rpcEndpoint), anchorWallet, {
+                preflightCommitment: 'recent',
```

Comment:
Better to use `confirmed` instead of `recent`, this way the account propagation will most likely be favorable.

Reply:
In this PR im only merging `dev` into `main`, im not implementing anything, if we can improve the code around this, better create an issue in github and implement the enhancements in a different PR 🙏🏼